### DISCUSSION TITLE
Enable email after docker installation

### DISCUSSION THREAD

**Discussion by jvmauricio** [firefly-iii/firefly-iii#8003] [Firefly III] [answered]
Good day everyone.

I setup firefly using docker straight from docker hub but did not setup email after installation.

what can I do to enable smtp after install? should I just install sendmail inside the container?

appreciate your help :)

**Comment by JC5** [comment, marked as answer]
No just set up the correct environment variables and you should be good to go.

**Reply by jvmauricio** [reply]
Thank you. I just re-run the container with the environment variables in Portainer and its now working.
